taskvine/src/manager/vine_current_transfers.c

```diff
@@ -119,35 +121,36 @@ void set_throttle(struct vine_manager *m, struct vine_worker_info *w, int is_des
 	}
 }
 
-int vine_current_transfers_set_failure(struct vine_manager *q, char *id)
+int vine_current_transfers_set_failure(struct vine_manager *q, char *id, const char *cachename)
 {
 	struct vine_transfer_pair *p = hash_table_lookup(q->current_transfer_table, id);
+	if (!p) {
+		return 0;
```

Comment:
Comment here: if there is no matching transfer record, then it means a worker failed and the record was removed b/c of `wipe_worker`.